HelloMixpanel/HelloMixpanel.xcodeproj/project.pbxproj

```diff
@@ -1913,7 +1854,7 @@
 				IPHONEOS_DEPLOYMENT_TARGET = 6.0;
 				ONLY_ACTIVE_ARCH = YES;
 				OTHER_CFLAGS = "";
-				PROVISIONING_PROFILE = "";
+				PROVISIONING_PROFILE = "b5fc6340-6933-4471-a458-b347298980ec";
```

Comment:
This probably will make it harder for people outside of Mixpanel to run sample app on devices because we don't have Mixpanel's provisioning profiles. It's nice to allow Xcode to choose it for us.